src/backend/distributed/commands/alter_table.c

```diff
@@ -1357,6 +1361,10 @@ List *
 GetViewCreationCommandsOfTable(Oid relationId)
 {
 	List *views = GetDependingViews(relationId);
+
+	/* we provide a notice message, as it could take a while to recreate the matviews */
+	NoticeMatViewCountToRecreate(views, relationId);
```

Comment:
Let's give this message if ErrorIfMatViewSizeExceedsTheLimit passes at the end of this function.